learning_resources/etl/utils.py

```diff
@@ -666,3 +668,20 @@ def transform_format(resource_format: str) -> list[str]:
 
     """
     return [RESOURCE_FORMAT_MAPPING[resource_format]]
```

Comment:
Unrelated to your PR but while testing this, the backpopulate_xpro_data task broke because a course had resource_format "Other" can we change it to 

```suggestion
        return [RESOURCE_FORMAT_MAPPING.get(resource_format, None)]

```

here? Or i can add a seperate pr

Reply:
Yeah, I had a conversation about this w/Peter last week, "Other" should not be an option for xpro, it's not valid.  But at the time, no course/program was using it.  As of this morning, one xpro course is using it, and I created a new issue about it: https://github.com/mitodl/hq/issues/4201

Reply:
For now I will change it to log an error and assign a default of "Online" if there is a `KeyError` in this PR.